python/BUSINESS_SUPPORT_OR_ABOVE_ENABLED/business-support-enabled_test.py

```diff
@@ -0,0 +1,163 @@
+import sys
+import unittest
+try:
+    from unittest.mock import MagicMock
+except ImportError:
+    import mock
+    from mock import MagicMock
+import botocore
+from botocore.exceptions import ClientError
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::::Account'
+
+#############
+# Main Code #
+#############
+
+CONFIG_CLIENT_MOCK = MagicMock()
+STS_CLIENT_MOCK = MagicMock()
+
+class Boto3Mock():
+    def client(self, client_name, *args, **kwargs):
+        if client_name == 'config':
+            return CONFIG_CLIENT_MOCK
+        elif client_name == 'sts':
+            return STS_CLIENT_MOCK
+        else:
+            raise Exception("Attempting to create an unknown client")
+
+sys.modules['boto3'] = Boto3Mock()
+
+RULE = __import__('business-support-enabled')
```

Comment:
Change to `BUSINESS_SUPPORT_OR_ABOVE_ENABLED`

Reply:
changed to `BUSINESS_SUPPORT_OR_ABOVE_ENABLED`